src/components/entity/list.vue

```diff
@@ -16,14 +16,7 @@ except according to the terms contained in the LICENSE file.
         <div class="form-group">
           <span class="icon-filter"></span>
         </div>
-        <div class="form-group">
-          <input v-model="searchTextbox" class="form-control search-textbox" :placeholder="$t('common.search')"
```

Comment:
Are we no longer using the `searchTextbox` data property? If so, can it be removed from `data()`?